### ISSUE TITLE
maxRange set to minRange in RemoveMargins() when channel_specifications is not null

### ISSUE THREAD

**Issue by phauchamps** [saeyslab/PeacoQC#12]
Hi,

When I try to use RemoveMargins() and set a non null channel_specifications, weird things happen: the maxRange parameter taken is equal to the provided minRange, not the provided maxRange. 

I am trying to fix this in a PR.

Thanks

Philippe

**Comment by AnneliesEmmaneel** [comment]
Hi @phauchamps,
I quickly checked this and for me it is working fine.
Can you show me how you are trying to use the channel_specifications parameter?
It should be a named list of lists like in the example of the function.
Kind regards,
Annelies

**Comment by phauchamps** [comment]
Hi @AnneliesEmmaneel ,
Thanks for looking into this !

Have you tested with channel_specifications having more than one channel ? In the example of the function, which works fine, only one channel is specified, and I think the problem arises when the named list (of lists) has more than one named element.

Attached is a reproduceable example.
[PeacoQC_Issue12.zip](https://github.com/saeyslab/PeacoQC/files/10557920/PeacoQC_Issue12.zip)

Kind regards,
Philippe

**Comment by AnneliesEmmaneel** [comment]
Hi @phauchamps,

I figured out what the issue is. It is not a list of lists that should be provided. It is actually a list of vectors.
If I changed your code (see below) it works perfectly. 

`
channelSpecs <- list( "450/50Violet-A" = c(0, 262143),
                      "670/30Yellow-A" = c(0, 20000))
`

I will change the documentation and add an additional example in the function.
Kind regards,
Annelies
